doc/howtos/bluetoothspeaker.rst

```diff
@@ -0,0 +1,141 @@
+Bluetooth Speaker
+#################
+
+How to setup your intel-iot-refkit image to function as a bluetooth speaker
```

Comment:
The instructions here only talk about the software components needed.  Isn't there some hardware needed too, such as an actual speaker?